Rename issues section - create issues (#218)
nebari-dev/nebari-docs: docs/docs/community/maintainers/triage-issues.md

---
title: Issue triaging
description: Guidelines and tips for maintaining Nebari
---

> Issue triage is a process by which Nebari maintainers and contributors intake and reviews new GitHub issues and requests,

Reviewer: ```suggestion
Issue triage is a process by which Nebari maintainers and contributors understand and review new GitHub issues and requests,
```

and organize them to be actioned. Triaging involves categorizing issues and Pull Requests based on factors such as priority,
area of ownership of the issue (tags), and the issue kind (bug, feature, and so on).

> 

Reviewer: What are your thoughts on including a goal for folks?

```suggestion

We aim to triage new issues or Pull Requests within 1-2 working days.
```

Author: agree!

Triage can happen asynchronously and continuously, or in regularly scheduled meetings.

> ## Why Is Triaging Beneficial?

Reviewer: ```suggestion
## Why Is triaging beneficial?
```
To use sentence case for headings. :)

Author: I kept mixing sentence and title case 😶‍🌫️  sorry
